x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/get_agent_marks.js

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
Let's make a note to review this file and its test along side of the UI code that will eventually use it.

Reply:
Not following :) What should the note be about? Is it because it's currently unused?